src/wazuh_qa_framework/system/wazuh_handler.py

```diff
@@ -623,6 +629,37 @@ def get_agent_id(self, host):
         """
         return self.wazuh_api.get_agent_id(host_ip=self.get_host_ansible_ip(host))
 
+    def get_agent_name_from_ip(self, agent_ip):
+        """Get agent name from ip.
+        Args:
+            agent_ip: Agent ip (str).
+        Returns:
+            str: Agent ip (str).
+        """
+        list_of_hosts = self.get_group_hosts()
+        for host in list_of_hosts:
+            if self.get_host_variables(host).get('ip') == agent_ip:
+                return self.get_host_variables(host).get('inventory_hostname_short')
+
+    def get_agents_id(self, manager, agents_list=None):
```

Comment:
Why agents_list is None by default?
```suggestion
    def get_agents_id(self, manager, agents_list):
```

Reply:
Done in https://github.com/wazuh/qa-system-framework/pull/71/commits/56b13fd995d0c2ff5bf72f28f07e8115eb52d246